File: app/src/release/java/org/tasks/DebugNetworkInterceptor.java
package org.tasks;

import com.google.api.client.http.HttpRequest;
import com.google.api.client.http.HttpResponse;
import javax.inject.Inject;
import okhttp3.OkHttpClient;

public class DebugNetworkInterceptor {
  @Inject
  public DebugNetworkInterceptor() {}

  public void add(OkHttpClient.Builder builder) {}

  public <T> T execute(HttpRequest httpRequest, Class<T> responseClass) {
    return null;
  }

  public <T> T report(HttpResponse httpResponse, Class<T> responseClass, long start, long finish) {
    return null;
  }
}

File: app/src/release/java/org/tasks/DebugNetworkInterceptor.kt
package org.tasks

import com.google.api.client.http.HttpRequest
import com.google.api.client.http.HttpResponse
import okhttp3.OkHttpClient
import javax.inject.Inject

class DebugNetworkInterceptor @Inject constructor() {
    fun apply(builder: OkHttpClient.Builder?) {}
    fun <T> execute(httpRequest: HttpRequest?, responseClass: Class<T>?): T? = null
    fun <T> report(httpResponse: HttpResponse?, responseClass: Class<T>?, start: Long, finish: Long): T? = null
}